src/main/java/jnr/constants/platform/darwin/TCP.java

```diff
@@ -1,14 +1,14 @@
 // WARNING: This file is autogenerated. DO NOT EDIT!
-// Generated 2010-07-31 18:57:13 +1000
+// Generated 2018-09-10 15:22:15 -0500
 package jnr.constants.platform.darwin;
 public enum TCP implements jnr.constants.Constant {
-TCP_MAX_SACK(3L),
```

Comment:
3L vs  4L

Reply:
Appears to have been manually added or something. I assume it didn't change across MacOS releases.

```
[] ~/projects/jnr-constants $ grep -r TCP_MAX_SACK /usr/include
/usr/include/netinet/tcp.h:#define	TCP_MAX_SACK	4	/* MAX # SACKs sent in any segment */
```